doc/integrator/filter.rst

```diff
@@ -119,12 +119,19 @@ Simple example:
 Step 2: Administration interface
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The final step is to assign the enumerated attributes to the right layer in the administration.
-Go to the ``metadata`` page, add a ``enumeratedAttributes`` to an existing layer.
-The value is a single string or a list of attributes (that we defined earlier in the ``vars_<project>.yaml`` file)
-separated with a comma.
+The final step is to assign the enumerateds or directed attributes to the right layer in the administration.
```

Comment:
I would say something like this, what do you think?:
"As a final step, if you wish to override the default behavior regarding how the GeoMapFish filter will provide filter attributes to the user, you can define this in the administration UI as follows."